lib/ansible/modules/cloud/azure/azure_rm_eventhub.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Fan Qiu, <fanqiu@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_eventhub
+version_added: "2.9"
+short_description: Manage Azure Event hub.
+description:
+    - Create, update and delete an Azure Event hub.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Event hub exists or will be created.
+        required: true
+    name:
+        description:
+            - Name of the Event hub.
+        required: true
+    state:
+        description:
+            - Assert the state of the Event hub. Use C(present) to create or update an event hub and C(absent) to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+    namespace:
+        description:
+            - The Namespace name
+        required: true
+    message_retention_in_days:
+        description:
+            - Number of days to retain the events for this Event Hub, value should be 1 to 7 days
+    partition_count:
+        description:
+            - Number of partitions created for the Event Hub, allowed values are from 1 to 32 partitions.
+            - Is not changeable after the creation
+    status:
+        description:
+            - Enumerates the possible values for the status of the Event Hub.
+        choices:
+            - active
+            - disabled
+            - send_disabled
+            - receive_disabled
+
+extends_documentation_fragment:
```

Comment:
is tags supported?

Reply:
No, according to the doc of python sdk, eventhub don't support tags. eventhub namespace support tags. I will check the details again over all files